examples/arm/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
this feels like a boilerplate code, are we sure we want everyone who wants to use selective build write this? That was my thought when I added a list variable at the top level so that some random build can also just "use it". If the answer is yes, I am ok with it too.

Reply:
I'm ok with users adding stuff like this.  It's a clean way for them to know what they are doing.